ethpm/_utils/chains.py

```diff
@@ -0,0 +1,102 @@
+import re
+from typing import Tuple
+from urllib import parse
+
+from eth_utils import add_0x_prefix, remove_0x_prefix, to_hex
+from web3 import Web3
+
+from ethpm.typing import URI
+
+
+def get_genesis_block_hash(web3: Web3) -> str:
```

Comment:
@kclowes maybe this makes sense to be a new API on the `w3.eth` module.  Something like `w3.eth.get_genesis_hash()`